### PR TITLE
Upload reports hint and automatically open reports file

### PR DESCRIPTION
**Pull request by BennyDioxide**
Since too many newbies asking question about crash without sending crash reports, I made a hint and make it automatically opens the two crash reports.
The idea was originally by @YutaYamamoto212.

### CODE REVIEW

File: lib/widget/dialog/GameCrash.dart [hunk 1]
```diff
@@ -31,7 +31,10 @@ class _GameCrashState extends State<GameCrash> {
                 textAlign: TextAlign.center,
                 style: const TextStyle(color: Colors.cyanAccent, fontSize: 20)),
             const SizedBox(height: 10),
-            Text(widget.errorLog, textAlign: TextAlign.center)
+            Text(widget.errorLog, textAlign: TextAlign.center),
+            const SizedBox(height: 10),
+            Text(I18n.format("log.game.crash.info"),
```
Comment (SiongSng) [inline lib/widget/dialog/GameCrash.dart:36]: The font is too big and looks weird

File: lib/screen/Log.dart [hunk 2]
```diff
@@ -317,6 +317,7 @@ class _LogScreenState extends State<LogScreen> {
           }
         }
       } else {
+        Util.openLatestCrashReport(instanceDir);
```
Comment (SiongSng) [inline lib/screen/Log.dart]: There should be a setting option to turn off auto open the crash report

File: assets/lang/zh_tw.json [hunk 3]
```diff
@@ -327,6 +327,7 @@
   "log.game.crash.title": "遊戲崩潰",
   "log.game.crash.code": "退出代碼",
   "log.game.crash.report": "遊戲崩潰報告",
+  "log.game.crash.info": "已自動開啟遊戲日誌及崩潰報告。\n請注意，退出代碼或錯誤訊息通常都不足以診斷問題！\n求助時，請一定要附上遊戲日誌及崩潰報告，以便診斷問題！",
```
Comment (SiongSng) [inline assets/lang/zh_tw.json:330]: 中文翻譯可以先不用填寫，會在 Crowdin 上進行翻譯

**Review by SiongSng — CHANGES_REQUESTED** [review]
Great feature, but some things need to be fixed

**Comment by SiongSng** [comment]
Good 我找時間 review

**Comment by BennyDioxide** [comment]
因為接下來的重寫，加上我最近沒時間
因此將此PR關閉？

**Comment by SiongSng** [comment]
OK

**Pull request closed without merging**